src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
Just because it's the way you what pfSense to behave does not make it good or even beneficial.  Microsoft does things because that's they way they want it to be too.  Doesn't necessarily make it better.  Doing that way just because that's the way we want it to be is not sound design.  Any good engineer will tell you that the more you "touch" something increases the probability of introducing an error/failure/etc.  The file is there, it's good.  Leave it alone and let it be.  Just the like CA bundle file.

If customers are willing to accept your inconsistently applied bloatware method of file integrity assurance.  That is between them and you.  But yes I prefer it be left alone once there.  And that's how it will be done on my systems.

If it really is that crittically important to you/pfSense then the rewriting on boot up is still not the "right"/"best" way of ensuring the content is correct, and has much more overhead then doing it in well engineered fashion that could also be applied to any file deemed to be critical.